st2api/Makefile

```diff
@@ -59,7 +59,7 @@ all: install
 
 install: wheelhouse changelog
 
-post_install:
+post_install: bdist_wheel
```

Comment:
Does latest version of pip ship with wheel support or we still need to explicitly install the wheel package?